package.json

```diff
@@ -32,16 +32,18 @@
     "@blackbaud/skyux": "^2.0.0"
   },
   "dependencies": {
-    "@blackbaud/skyux-builder-plugin-stache": "~1.7.0",
+    "@blackbaud/skyux-lib-clipboard": "1.0.0-rc.1",
+    "@blackbaud/skyux-lib-code-block": "1.0.0-rc.1",
+    "@blackbaud/skyux-builder-plugin-stache": "https://github.com/blackbaud/skyux-builder-plugin-stache.git#feature-sky-code-block",
     "@blackbaud/skyux-builder-stache-search": "^1.3.0",
     "btoa": "1.1.2",
     "es6-object-assign": "1.1.0",
     "lodash.get": "4.4.2",
     "prismjs": "1.15.0"
   },
   "devDependencies": {
-    "@blackbaud/skyux": "2.30.0",
-    "@blackbaud/skyux-builder": "1.29.0",
-    "@blackbaud/skyux-lib-testing": "1.0.0"
+    "@blackbaud/skyux": "2.33.0",
```

Comment:
We had a hiccup in our release schedule. The latest version is actually `2.33.1`.